src/index.js

```diff
@@ -1,21 +1,44 @@
 import 'isomorphic-fetch';
+import promiseRetry from 'promise-retry';
 import objectAssign from 'object-assign';
 
-export default function jsonFetch(URL, fetchOptions = {}) {
-  let fetchOptionsClone = objectAssign({}, fetchOptions);
+export {retriers} from './retriers';
+
+const DEFAULT_RETRY_OPTIONS = {
+  retries: 0,
+  shouldRetry: () => false
+};
+
+export default function jsonFetch(URL, options = {}) {
+  let fetchOptions = objectAssign({}, options);
```

Comment:
You bet! I need to look at the best practices again -- do we prefer consts when fields on an object might change?